src/main/java/duke/exception/DukeInvalidAddTaskException.java

```diff
@@ -0,0 +1,4 @@
+package duke.exception;
+
+public class DukeInvalidAddTaskException extends Exception {
+}
```

Comment:
You may want to use inheritance to give a more meaningful error message in the custom exception you have created.